recipe/meta.yaml

```diff
@@ -23,6 +23,7 @@ requirements:
     - {{ compiler('cuda') }}
     - cmake
     - ninja
+    - pennylane >=0.33
   host:
```

Comment:
Think we want to add this to `host`. Though maybe I'm missing something

```suggestion
  host:
    - pennylane >=0.33
```

Reply:
I think we can remove that too, my bad.

Reply:
All good. Just trying to follow the changes here :)